assets/xml/textures/nes_font_static.xml

```diff
@@ -14,130 +14,131 @@
-        <Texture Name="gMsgCharLowercaseDiaeresisATex" OutName="msg_char_lowercase_diaeresis_a" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharLowercaseLatinScriptCTex" OutName="msg_char_lowercase_latin_script_c" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharLowercaseGraveAccentedETex" OutName="msg_char_lowercase_grave_accented_e" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharLowercaseAccentedETex" OutName="msg_char_lowercase_accented_e" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharLowercaseCircumflexedETex" OutName="msg_char_lowercase_circumflexed_e" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharLowercaseDiaeresisETex" OutName="msg_char_lowercase_diaeresis_e" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharLowercaseDiaeresisITex" OutName="msg_char_lowercase_diaeresis_i" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharLowercaseCircumflexedOTex" OutName="msg_char_lowercase_circumflexed_o" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharLowercaseDiaeresisOTex" OutName="msg_char_lowercase_diaeresis_o" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharLowercaseGraveAccentedUTex" OutName="msg_char_lowercase_grave_accented_u" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharLowercaseCircumflexedUTex" OutName="msg_char_lowercase_circumflexed_u" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharLowercaseDiaeresisUTex" OutName="msg_char_lowercase_diaeresis_u" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharAButtonTex" OutName="msg_char_a_button" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharBButtonTex" OutName="msg_char_b_button" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharCButtonTex" OutName="msg_char_c_button" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharLTriggerTex" OutName="msg_char_l_trigger" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharRTriggerTex" OutName="msg_char_r_trigger" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharZTriggerTex" OutName="msg_char_z_trigger" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharCUpButtonTex" OutName="msg_char_c_up_button" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharCDownButtonTex" OutName="msg_char_c_down_button" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharCLeftButtonTex" OutName="msg_char_c_left_button" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharCRightButtonTex" OutName="msg_char_c_right_button" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharTildeTex" OutName="msg_char_tilde" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharNoBreakSpaceTex" OutName="msg_char_no_break_space" Format="i4" Width="16" Height="16"/>
```

Comment:
I think this character's name is `DEL` [according to unicode](https://www.compart.com/en/unicode/U+007F).
```suggestion
        <Texture Name="gMsgCharDelTex" OutName="msg_char_del" Format="i4" Width="16" Height="16"/>
```